components/bridge/AddToken.tsx

```diff
@@ -0,0 +1,204 @@
+            exit={{ opacity: 0 }}
+            className="fixed top-0 left-0 w-full h-full bg-black bg-opacity-60 z-50 flex"
+            id="modal-bg"
+          >
+            <motion.div
+              initial={{ opacity: 0, top: "-100%" }}
+              animate={{ opacity: 1, top: "50%" }}
+              exit={{ opacity: 0, top: "-100%" }}
+              transition={{
+                duration: 0.3,
+              }}
+              className="bg-white w-[548px] top-[50%] left-[50%] translate-x-[-50%] translate-y-[-50%] absolute px-[60px] py-[50px] flex flex-col items-start overflow-y-auto z-50 md:hidden justify-center rounded-[20px]"
+            >
+              <div className="w-full">
+                <img
+                  src={cross.src}
+                  alt="close"
+                  className="cursor-pointer h-7 ml-auto translate-x-[150%] translate-y-[-100%]"
+                  onClick={() => {
+                    dispatch(toggleAddTokenToast(false));
+                  }}
+                />
+              </div>
+              <span className="text-3xl font-bold -mt-7">
+                Add token to wallet
+              </span>
+              <span className="text-sm font-normal text-[#4d4d4d] mt-3">
+                After completing a transaction, on the Fuse network you will
+                receive a version of the token minted specifically for Fuse
+                Bridge. You can add it to your wallet manually using the
+                contract address, or simply click Add token button below.
+              </span>
+              <div className="px-10 py-5 rounded-[14px] border-[1px] w-full border-[#D2D5D8] mt-6 flex items-center justify-between">
+                <img
+                  src={getTokenIcon(tokenSlice.token as string)}
+                  alt="token"
+                  className="w-[10%]"
+                />
+                <span className="text-sm font-semibold ml-2 w-[20%]">
+                  {tokenSlice.token}
+                </span>
+                <span className="text-base text-[#666] w-[60%] break-all">
+                  {getTokenAddress(tokenSlice.token as string)}
+                </span>
+                <div className="relative w-[10%]">
+                  <Image
+                    src={copy}
+                    alt="copy"
+                    className="h-4 w-[100%] cursor-pointer"
+                    onClick={() => {
+                      handleCopy(
+                        getTokenAddress(tokenSlice.token as string) as string
+                      );
+                    }}
+                  />
+                  <div
+                    className={`copy-tooltip absolute left-1/2 -translate-x-1/2 top-8 bg-black px-[13.4px] py-[7.56px] rounded-2xl w-max shadow-lg text-white text-sm leading-none font-medium ${
+                      isCopied ? "block" : "hidden"
+                    }`}
+                  >
+                    <p>{"Address copied"}</p>
```

Comment:
@MayankMittal1 you can use [Copy component](https://github.com/fuseio/console-dapp/blob/master/components/ui/Copy.tsx)

Reply:
@MusabShakeel576 fixed